cra-tf-validate-ignore-goals.json

```diff
@@ -0,0 +1,41 @@
+{
+    "description": "The below ignore file is specifically for the patterns/roks solution",
+    "scc_goals": [
+        {
+            "scc_goal_id": "3000107",
+            "description": "Check whether Cloud Object Storage network access is restricted to a specific IP range",
+            "ignore_reason": "Tracking in https://github.ibm.com/GoldenEye/issues/issues/4332",
+            "is_valid": true
+        },
+        {
+            "scc_goal_id": "3000114",
+            "description": "Check whether Cloud Object Storage buckets are enabled with IBM Activity Tracker",
+            "ignore_reason": "This goal is not an FSCloud required goal",
+            "is_valid": false
+        },
+        {
+            "scc_goal_id": "3000115",
+            "description": "Check whether Cloud Object Storage buckets are enabled with IBM Cloud Monitoring",
+            "ignore_reason": "This goal is not an FSCloud required goal",
+            "is_valid": false
+        },
+        {
+            "scc_goal_id": "3000116",
+            "description": "Check whether Cloud Object Storage bucket resiliency is set to cross region",
+            "ignore_reason": "Tracking in https://github.ibm.com/GoldenEye/issues/issues/4333",
+            "is_valid": true
+        },
+        {
+            "scc_goal_id": "3000452",
+            "description": "Check whether Virtual Private Cloud (VPC) network access control lists don't allow egress from 0.0.0.0/0 to any port",
+            "ignore_reason": "Tracking in https://github.ibm.com/goldeneye/issues/issues/3759",
+            "is_valid": true
+        },
+        {
+            "scc_goal_id": "3000408",
+            "description": "Check whether Flow Logs for VPC are enabled",
+            "ignore_reason": "Flow logs are enabled, so seems to be CRA bug. Tracking in https://github.ibm.com/oneibmcloud/CD-CRA/issues/1779",
+            "is_valid": false
+        }
+    ]
+}
```

Comment:
You need to create a new file called `cra-tf-validate-ignore-rules.json` and delete this file. This file is referencing SCC v1 goals which are now replaced by SCC rules.

Reply:
can you point to a repo ? because the landing zone is also using the same file

Reply:
Here is an example https://github.com/terraform-ibm-modules/terraform-ibm-icd-postgresql/blob/main/cra-tf-validate-ignore-rules.json however you will have different rule IDs that you may need to ignore. You would have to run CRA manually to see if you need to skip any failing rules

Reply:
Deleted the file